Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -1009,7 +1009,7 @@ void KeyframeEffect::computeCSSAnimationBlendingKeyframes(const RenderStyle& una
     auto cssAnimation = downcast<CSSAnimation>(animation());
     auto& backingAnimation = cssAnimation->backingAnimation();
 
-    KeyframeList keyframeList(backingAnimation.name().string);
+    KeyframeList keyframeList(AtomString { backingAnimation.name().string });
```

Comment:
Maybe these backing animation names should be `AtomString`?

Reply:
It is possible but I am not sure so I didn't make the change as part of this large patch.